src/services/dbServices.js

```diff
@@ -63,6 +63,32 @@ const DbServices = {
    */
   bulkCreate(model, data) {
     return model.bulkCreate(data, { returning: true });
+  },
+
+  /**
+   * @param {object} model primary model /table
+   * @param {string} id the id of primary model /table
+   * @param {object} table2 object containing 3 properties for the secondary model /table
+   * @param {object} table2.model secondary model /table
+   * @param {string} table2.alias alias of secondary model /table
+   * @param {object} table2.column condition that affects how data is returned from secondary table
+   * @returns {Promise} Promise resolved or rejected
+   * @description gets data from a primary table and also joins data from a secondary table
+   *
+   * @example
+   *   getTwoTables(Request, 1, { model: User, alias: 'User', column: { lineManager: managerId } })
+   */
+  getTwoTables(model, id, table2) {
```

Comment:
change ` getTwoTables` to `findOneIncludeModel`  and table2 to `modelDetailsToInclude` or `findOneIncludeTable` and `tableDetailsToInclude`.  it makes devs to easily read your code.

Reply:
Done